bin/setup

```diff
@@ -12,13 +12,8 @@ Dir.chdir APP_ROOT do
   system "gem install bundler --conservative"
   system "bundle check || bundle install"
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?("config/database.yml")
-  #   system "cp config/database.yml.sample config/database.yml"
-  # end
-
   puts "\n== Preparing database =="
-  system "bin/rake db:setup"
+  system "bin/rake db:setup permissions:load db:populate_test"
```

Comment:
Just for new developers, so you run `bin/setup` and then you can start the server.